### ISSUE TITLE
Tagging release

### ISSUE THREAD

**Issue by dvershinin** [ukarim/ngx_markdown_filter_module#1]
Hello, nice work on the module. Is it stable? If it is, please tag a release version.

**Comment by ukarim** [comment]
@dvershinin  yep. I have nginx-1.20.2 installation with this module included running for almost 1year. And it works just fine, without any issues.
I just applied one patch that properly set content-length header (previous version erased content-length header that led to chuncked transfer encoding).

Created tag 0.1.0
